packages/@adobe/spectrum-css-temp/components/tags/index.css

```diff
@@ -49,18 +51,17 @@ governing permissions and limitations under the License.
     margin-inline-end: var(--spectrum-tag-icon-padding-x);
     margin-inline-start: calc(var(--spectrum-tag-deletable-border-size-key-focus) * -1);
   }
+}
 
-  .spectrum-Tags-itemClearButton {
-    display: flex;
-    margin-inline-end: calc(-0.5 * var(--spectrum-tag-padding-x));
-    margin-inline-start: var(--spectrum-tag-deletable-border-size-key-focus);
-    block-size: var(--spectrum-alias-font-size-default);
-    inline-size: var(--spectrum-alias-font-size-default);
-    border-radius: var(--spectrum-alias-border-radius-small);
-  }
+.spectrum-Tag-icon {
+  grid-area: icon;
+  align-items: center;
+  justify-items: center;
```

Comment:
`spectrum-Tag-icon` isn't display flex or grid, why are these two lines needed?

Reply:
i may have copied it from somewhere...removing